File: kimhyunji/PGS/Level 1/콜라 문제.java
class Solution {
    public int solution(int a, int b, int n) {
        int answer = 0;
        
        while(n>=a){ //콜라가 마트 조건보다 많을 때
            int k = n%a; //마트에 바꾸고 남은 콜라
            n = (int)(n/a)*b; //빈병을 b개의 콜라병으로 바꾸기
            
            answer += n; //바꾼 콜라병 수 더하기
             
            n += k; //남아있던 콜라 더하기
        }
        
        return answer;
    }
}

File: baekyeonsu/콜라 문제.js
function solution(a, b, n) {
  let answer = 0;
  let current = n;

  while (current >= a) {
    answer += parseInt(current / a, 10) * b;
    current = parseInt(current / a, 10) * b + parseInt(current % a, 10);
  }

  return answer;
}
